lib/LightPanels/LightPanels.js

```diff
@@ -247,6 +256,21 @@ module.exports = kind(
 	*/
 
 	/**
+	* Changes to the specific index, accepting a number of additional options.
+	*
+	* @param {Number} to - The index of the panel we wish to transition to.
+	* @param {module:enyo/LightPanels~UpdateIndexOptions} [opts] - Additional options to be used
+	*	when updating the panel index.
+	* @param {Number} [from] - If known, the index we are transitioning from.
+	* @public
+	*/
+	updateIndex: function (to, opts, from) {
```

Comment:
I wanted to consolidate the logic from `indexChanged` so it would be in one location, but resulted in slightly awkward ergonomics for `updateIndex`. That being said, only the internal `indexChanged` would realistically utilize the third parameter (`from`). Let me know if this is too awkward in general and we can move to use an `updateIndexDirect` method, which I was trying really hard to avoid. :smile: